### DISCUSSION TITLE
Setting colours of cells

### DISCUSSION THREAD

**Discussion by Jackkkkk494** [Akascape/CTkTable#59] [Q&A] [answered]
Currently making an algorithm which checks inventory levels in my database and assigns this value a colour, I want this to change the colour of the cells in accordance to the row and column, is this possible?

**Comment by Akascape** [comment, marked as answer]
@Jackkkkk494 When you insert a value in a cell using `.insert` method, you can pass `fg_color` parameter too. Or you can simply set the hover color to the cell using `.select(row, column)` method. 

If you don't want to change any value in a cell but want to only change color:
```python
table.insert(row=row, column=column, value=table.get(row, colomn), fg_color="color_name")
```

**Reply by aeris07** [reply]
Hello, @Akascape , and thank you for this awesome library.

I think I'm missing something.
I'm trying to change color of any cell, at this sample code as you show, but getting error as below.

```
root = customtkinter.CTk()
frame = customtkinter.CTkFrame(root)
frame.pack(expand=True, fill="both")
value = [[1, 2, 3, 4, 5],
         [1, 2, 3, 4, 5],
         [1, 2, 3, 4, 5],
         [1, 2, 3, 4, 5],
         [1, 2, 3, 4, 5], ]
table = CTkTable(master=frame, row=5, column=5, values=value, height=100)

table.pack(expand=True, fill="both", padx=20, pady=20)
table.insert(row=2, column=1, value=table.get(2, 1), fg_color="blue")
```
......CTkTable\ctktable.py", line 395, in get
    return self.data[row,column]["values"]
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
KeyError: 'values'

**Reply by Akascape** [reply]
@aeris07 are you using the latest version?

**Comment by aeris07** [comment]
I didn't notice my version,
Updating to latest version(0.7 to 0.8) fixed it.
Thank you so much.

Akash Bora ***@***.***>, 6 Ara 2023 Çar, 07:34 tarihinde şunu
yazdı:

**Comment by aeris07** [comment]
Hello @Akascape/CTkTable
***@***.***>
Thank you for your helpful and quick response,
Great, it worked like a charm,
